lms/templates/courseware/courseware-chromeless.html

```diff
@@ -53,7 +56,7 @@
 
   <%static:js group='courseware'/>
 
-  <%include file="../discussion/_js_body_dependencies.html" />
+  <%include file="/mathjax_include.html" args="disable_fast_preview=True"/>
```

Comment:
@cahrens Yep!  We had a [copy-paste error in our previous pull request](https://github.com/edx/edx-platform/pull/13483/files#diff-38172c55444ae942f6d8e53dd83cf665R59), which referenced the template variable `disable_fast_preview` which is [set to True in `_js_body_dependencies.html`](https://github.com/edx/edx-platform/blob/master/lms/templates/discussion/_js_body_dependencies.html#L5).  

That variable is not defined in the updated `courseware*.html` templates, and so the Fast Preview feature was enabled for Courseware, causing timing issues.  So we've set it to `True`, and all looks good again.